src/plugins/vis_type_timeseries/public/application/components/panel_config.js

```diff
@@ -67,7 +67,9 @@ export function PanelConfig(props) {
     return (
       <FormValidationContext.Provider value={updateControlValidity}>
         <VisDataContext.Provider value={visData}>
-          <Component {...props} />
+          <div data-test-subj={`tvbPanelConfig__${model.type}`}>
```

Comment:
This wrapper just replaces explicit `div` element in each panel config from `timeseries`, `metric`, `gauge`, etc.